factfinder/calculate.py

```diff
@@ -150,34 +151,18 @@ def calculate_e_m_median(self, pff_variable: str, geotype: str) -> pd.DataFrame:
             index="census_geoid", columns="pff_variable", values=["e"]
         )
 
-        # Empty dataframe to store the results
-        results = pd.DataFrame()
+        def get_median_and_median_moe(ranges, row, DF):
+            md = Median(ranges, row, DF)
+            e = md.median
+            m = md.median_moe
+            return pd.Series({'e': e, 'm': m})
+
+        results = df_pivoted.e.apply(lambda x: get_median_and_median_moe(
+            ranges, x, DF=design_factor), axis=1)
```

Comment:
@mgraber  
I did the same thing in the current branch. this is a lot easier

Reply:
#190

Reply:
Ah, I missed this. Thanks!